magenta/review/summary_generation_sequences.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Since this isn't a jekyll blog post, I think you can skip this yaml frontmatter.